conference/talks.py

```diff
@@ -0,0 +1,61 @@
+from django.conf.urls import url
+from django.shortcuts import get_object_or_404
+from django.template.response import TemplateResponse
+
+from conference.models import Talk
+
+
+def talk(request, talk_slug):
+    """
+    Display Talk
+    """
+    talk = get_object_or_404(Talk, slug=talk_slug)
+    talk_as_dict = dump_relevant_talk_information_to_dict(talk)
+
+    return TemplateResponse(
+        request,
+        "ep19/bs/talks/talk.html",
+        {"title": talk.title, "talk": talk, "talk_as_dict": talk_as_dict},
```

Comment:
Do we need to introduce the talk in dict form here? Looking at the temaplate, it uses `talk.speaker_set.user.assopy_user.name()`; how about we add a `Speaker.name` property, so that we can iterate over `talk.speaker_set` in the template directly?

Reply:
I just reused the view from Preview, but later moved to the template based on voting, hence the mismatch... will fix